recipes/elegansbot/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "elegansbot" %}
+{% set version = "1.0.2" %}
+{% set python_min = "3.8" %}
```

Comment:
conda-forge officially only supports python >=3.9, so you should drop specifying `python_min` here and just use the global pinning of 3.9